scripts/install_third_party.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
@seanlip why do we use this method instead of using pip to install the dependencies locally from a similar metadata file? I see that this is trying to [duplicate](https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-t) what `pip install --t` does.